app/components/common/error/index.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+angular.module('odca.error', [])
+  .config(['$httpProvider', function ($httpProvider) {
+    $httpProvider.interceptors.push('errorHttpInterceptor');
+  }])
+  .run(['$window', 'error', function ($window, error) {
+    $window.onerror = onError;
```

Comment:
Not sure what you mean, the `onerror`? I'm just hooking into the browser's [global error handler](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onerror).